### ISSUE TITLE
Rename QTCBS to QTBC11 

### ISSUE THREAD

**Issue by PDuckworth** [strands-project/strands_qsr_lib#235]
As per [here](http://matwbn.icm.edu.pl/ksiazki/cc/cc35/cc3516.pdf) or [here](http://www.sciencedirect.com/science/article/pii/S0957417410011863), QTCBS is already defined as QTCB11.

Similar issue may exist with QTCC also. @yianni @cdondrup

**Comment by cdondrup** [comment]
Yes it does also exist for QTCC. As I said, if this really is an issue, please feel free to refactor and also refactor strands_hri as this is using it heavily. I have no time to do this so please don't break my code just because of a naming convention.

**Comment by marc-hanheide** [comment]
@PDuckworth you are right, but as this is mostly a documentation issue rather than a code issue, I suggest to simply update the documentation and point out the equivalence there.

**Comment by marc-hanheide** [comment]
Or, alternatively introduce aliases in QSRlib to avoid breaking existing code.

**Comment by yianni** [comment]
@marc-hanheide Not sure what you mean by aliases, but if you are referring to dictionary mapping it might be tricky as this name is unique and tightly integrated to the retrieval and parsing of the QSRs returned also.

@cdondrup Don't worry about breaking code. Will not actually refactor but do one of the following solutions I see viable and quick:

* Copy the current QTC?S and rename them, and documentation will only refer to the renamed one, hence leaving STRANDS code unbroken but not publicising it outside this naming. Only issue I see that any change made will mean to actually copy the files to the other.
* Might be more elegant to subclass QTC?S from the proper naming conventions and simply overwrite the name. Will have to check the inheritance dependencies to decide how quickly this can be done.

@PDuckworth @cdondrup What about QTCCS and QTCBCS? To what should they be renamed?

**Comment by cdondrup** [comment]
The versions used are QTCB11, QTCC21, and then these are combined into QTCBC which would then be QTCB11C21 to be exact but I defined that as QTCBC in my paper. It's up to you how you want to name them.
